src/sailor/db/elastic_model.lua

```diff
@@ -0,0 +1,50 @@
+local elastic = require "sailor.db.elasticsearch"
+local Emodel = {}
+
+
+Emodel.new = function(model_name)
+	
+	local self = {}
+	local ob = require("models."..model_name)
+	Etype = ob.type
+	keys = ob.keys	
+	local parameters={}
+	-- This method should be used when you are saving the parameters. If second parameter is present,
+	-- it will write that value into ES
+	self.save = function(id, ...)
+		if ... ~= nil and type(...) == "table" then 
+			return elastic.index(Etype, id , ...)
+		else
+			return elastic.index(Etype, id, parameters)
+		end
+	end
+
+	-- Get a document using ID
+	self.get = function(id)
+		return elastic.get(Etype, id)
+	end
+	-- Delete a document using ID
+	self.delete = function(id)
+		return elastic.delete(Etype,id)
+	end
+	self.search = function(word)
+		return elastic.search(Etype, word)
```

Comment:
I guess that we can search for than one single word. We usually call it query in the search world. But if query conflicts with the notion of queries in relational databases, you can call them `terms` (it is still not correct as people can have something like `+fox -quick`).